README/ReleaseNotes/v626/index.md

```diff
@@ -37,12 +37,14 @@ The following people have contributed to this new version:
  Wouter Verkerke, NIKHEF/Atlas,\
  Stefan Wunsch, CERN/SFT
 
-## Deprecation and Removal
+## Deprecation, Removal, Backward Incompatibilities
 
 - The "Virtual MonteCarlo" facility VMC (`montecarlo/vmc`) has been removed from ROOT. The development of this package has moved to a [separate project](https://github.com/vmc-project/). ROOT's copy of VMC was deprecated since v6.18.
 
 - `TTreeProcessorMT::SetMaxTasksPerFilePerWorker` has been removed. `TTreeProcessorMT::SetTasksPerWorkerHint` is a superior alternative.
 
```

Comment:
While you're at it :-) can you remove the empty lines in this enumeration? You can see here https://root.cern/doc/master/release-notes.html#deprecation-and-removal that it adds extra vertical space for the bullet point. (This is also where your change will show up once merged!)

```suggestion
- `TTreeProcessorMT::SetMaxTasksPerFilePerWorker` has been removed. `TTreeProcessorMT::SetTasksPerWorkerHint` is a superior alternative.
```